skimage/draw/draw.py

```diff
@@ -693,3 +693,57 @@ def bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape=None):
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
     """
     return _bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape)
+
+
+def rectangle(origin, extent, shape=None):
+    """Generate coordinates of pixels within an nd-rectangle.
+
+    Parameters
+    ----------
+    origin : tuple
+        origin of the rectangle, (page,row,column,channel)
+    extent : tuple
+        extent (num_pages,num_rows,num_cols,num_chanels..) of the rectangle.
```

Comment:
Hi again, 
One more thing - I was thinking about drawing an enclosing volume and got me thinking about convex hulls... which naturally took me to the scipy.spatial module. I see there is a hyperrectangle class  in that module, though it could use some documentation and does not have a draw method. Maybe a better thing would be to include this as a draw method on that class? The problem is, I'm not sure if anyone would know to look there if they were looking to draw on an image. Anyway, this is above my pay grade -- this was actually my first PR - I got a chance to get my feet wet at the scipy sprints so whatever you suggest I'm just happy to get a chance to contribute and learn the ropes....